addons/mail/static/src/messaging/messaging_env.js

```diff
@@ -28,6 +29,7 @@ async function addMessagingToEnv(env) {
         autofetchPartnerImStatus: true,
         disableAnimation: false,
         entities: undefined,
+        entityManager: undefined,
```

Comment:
I think those 2 keys should be merged.

`entityManager` should probably be responsible of the `entities`, and it should probably call `_addMessagingEntities` (can be renamed to just `_addEntities` since there is no kind of non messaging entity) / `generateEntities` itself. Maybe even most of entity core should be there too.